docs/azure/migration/appcat/dotnet-cli.md

```diff
@@ -0,0 +1,94 @@
+
+    > [!NOTE]
+    > If this is your first time running application and code assessment for .NET, you will see an informational message about telemetry and how to opt-out if you should want to.
+
+1. A screen is presented that allows you to pick the projects in your solution to analyze. Use the arrow keys to highlight individual projects and press the spacebar to select them. Press **Enter** when you're ready.
+    ![A screenshot of the CLI tool's project selection](./media/cli/cli-analyze-project-selection.png)
+1. Next you'll be presented with the option to analyze **Source code and settings**, **Binary dependencies**, or both. Make your choice and press **Enter**.
+    ![Screenshot of the CLI dependency selection screen](./media/cli/cli-select-dependencies.png)
+    > [!NOTE]
+    > The **source code and settings** option will only scan the source code in the projects you selected on the previous screen. The **Binary dependencies** option will scan any dependencies (such as NuGet packages) your projects rely on. You can expect to see many more issues identified when **binary dependencies** is selected.
+    > You should consider selecting only **source code and settings** first to figure out what issues reside in your code and fix those before runnig the **binary dependencies** scan.
+    >
+1. You'll then be prompted to generate a report with the results of the analysis. The output can be formatted as CSV, HTML, or JSON. Press **Enter**.
+    ![Screenshot of the CLI report format selection screen](./media/cli/cli-report-format-selection.png)
+1. You'll be prompted for a report name. Input the name and press **Enter**.
+    ![Screenshot of the CLI report name screen](./media/cli/cli-report-name.png)
+1. Finally, you'll be asked whether you want to perform the scan. Press **y** to continue, or **n** to go back and change options.
+1. Once the analysis completes, the report is saved, and a summary of the results are displayed.
+    ![Screenshot of the analysis summary](./media/cli/cli-scan-finished.png)
+
+## Interpret the results
+
+For the purposes of explaining the results of the analysis, we're going to use the HTML report. The report presents its results in a dashboard format. In this section we'll describe the different sections of the dashboard and how to interpret them.
+
+### Dashboard view
+
+The main dashboard shows a summary of the results of the scan. It shows the number of projects scanned, the number of issues found, it estimates the amount of effort to fix the issues in story points, and it classifies the issues by severity from Mandatory to Informational.
+
+Here are the issue severity classifications:
+
+* **Mandatory** - the issue has to be resolved for the migration to be successful.
+* **Optional** - the issue discovered is a real issue and fixing it could improve the app after migration, however it is not blocking.
+* **Potential** - we are not sure if it is necessarily a blocking problem, but raised just in case.
```

Comment:
```suggestion
* **Potential** - it could or could not be a blocking problem depending on the specifics of your app and the migration scenario, so the tool draws your attention and suggests what checks could be made to ensure the application will work in Azure.
```